model/task/db.go

```diff
@@ -1826,6 +1826,7 @@ func updateAllTasksForAllMatchingDependencies(ctx context.Context, taskIDs []str
 					"else": false,
 				}}},
 			},
+			addDisplayStatus,
```

Comment:
Is this an expensive stage to calculate? I'm a little bit concerned about how many places we're adding this to. This is already a pretty expensive update I think, for example. Did Kim discuss this at all?